src/languages/es.js

```diff
@@ -486,7 +486,7 @@ export default {
         error: {
             invalidFormatEmailLogin: 'El email introducido no es válido. Corrígelo e inténtalo de nuevo.',
         },
-        cannotGetAccountDetails: 'No pudimos cargar los detalles de tu cuenta, por favor intenta iniciar sesión de nuevo.',
+        cannotGetAccountDetails: 'No se pudo los detalles de tu cuenta, por favor intenta iniciar sesión de nuevo.',
```

Comment:
As in slack, should it be `No se pudieron cargar los detalles de tu cuenta.`?
That also looks like a good translation